.vuepress/config.js

```diff
@@ -162,6 +165,7 @@ module.exports = {
                 collapsable: true,
 				sidebarDepth: 1,
                 children: [
+                    '/config-HEDT/nahalem',
```

Comment:
```suggestion
                    '/config-HEDT/nehalem',
```

Reply:
Oh thank you, the file name itself is botched so will need to resolve that as well